package/gluon-mesh-vpn-wireguard/README.md

```diff
@@ -0,0 +1,38 @@
+# gluon-mesh-vpn-wireguard
+
+This package will allow wireguard [1] to be used in gluon. Wireguard being a
+VPN on OSI layer 3 is useful for mesh protocols that operate on layer 3.
+
+[1] https://wireguard.io
+
+## including servers via site.conf
+This is similar to the fastd-based mesh_vpn structure.
+
+```
+          mesh_vpn = {
+                  mtu = 1374,
+
+                  wireguard = {
+                          enabled = true,
+                          groups = {
+                                  backbone = {
+                                          limit = '2', -- currently unused
+                                          peers = {
+                                                  gw02 = {
+                                                          enabled = true,
+                                                          key = 'bog2DzyiC0Os7y1GloEw0afb8bLdZ9SzVQCd44Eock4=',
+                                                          remote = 'gw02.babel.ffm.freifunk.net',
+                                                          broker_port = 40000,
+                                                  },
```

Comment:
What about persistent-keepalive? I think it's needed for stupid NATs.

Reply:
what about it? I take you would like persistent-keepalive be activated. How do we do that on the wg connection?

I think it would be a good idea to have something ensuring NAT compatibility even despite babeld and mmfd are causing occasional traffic on each VPN link.

Reply:
what is a reason to have this configured with many different wg calls? I feel it is better to hand the config to wireguard in one step. An alternative could be to use config files, but then we'd have a file sitting around somewhere that would serve no useful purpose.... Frankly I like it more in its current form.

... and I added persistent-keepalive.